sources/osg/Matrix.js

```diff
@@ -358,21 +369,17 @@ define( [
             return r;
         },
 
-        makeLookAt: ( function () {
-            var f = [ 0.0, 0.0, 0.0 ];
+        makeLook: ( function () {
```

Comment:
ok.
(it's about not recompute direction when we arleady have it, and don't have a target, like in the light case)